### PR TITLE
adding note about grafana-dev instance

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/stolostron/backlog/issues/25815

### CODE REVIEW

**Review by marcolan018 — APPROVED** [review]
lgtm

**Review by oafischer — COMMENTED** [review]
The content looks good, thanks for implementing it. I do see your point about this being more of a known issue and would agree that it belongs there. Since we also try to avoid special tags, moving it there would help us save one more **Note:** as well.

**Comment by dockerymick** [comment]
@oafischer @marcolan018 May you all re-review the PR

File: observability/design_grafana.adoc [hunk 1]
```diff
@@ -10,7 +10,11 @@ You can design your Grafana dashboard by creating a `grafana-dev` instance.
 [#setting-up-the-grafana-developer-instance]
 == Setting up the Grafana developer instance
 
-First, clone the https://github.com/stolostron/multicluster-observability-operator[`stolostron/multicluster-observability-operator/`] repository, so that you are able to run the scripts that are in the `tools` folder. Complete the following steps to set up the Grafana developer instance:
+First, clone the https://github.com/stolostron/multicluster-observability-operator[`stolostron/multicluster-observability-operator/`] repository, so that you are able to run the scripts that are in the `tools` folder. 
+
+*Note*: Be sure to use most current `grafana-dev` instance.
```
Comment (oafischer) [inline observability/design_grafana.adoc]: Do we need to keep this note? Or can we just integrate it into the intro paragraph?

Comment (dockerymick) [reply observability/design_grafana.adoc]: Great idea!

**Comment by dockerymick** [comment]
@oafischer updated

File: observability/design_grafana.adoc (hunk 1 above)
Comment (oafischer) [reply observability/design_grafana.adoc]: I think there's also a missing "the" in there (the most current), otherwise looks good. I'll approve, thanks!

**Review by oafischer — APPROVED** [review]

**Pull request merged**